framework/flamingo/sessions.go

```diff
@@ -57,9 +58,7 @@ func (m *SessionModule) Inject(config *struct {
 	m.storeLength = int(config.StoreLength)
 	m.maxAge = int(config.MaxAge)
 	m.path = config.Path
-	m.redisHost, m.redisPassword = getRedisConnectionInformation(config.RedisURL, config.RedisHost, config.RedisPassword)
-	m.redisIdleConnections = int(config.RedisIdleConnections)
```

Comment:
Did we loose this by mistake?

Reply:
Yes, you are right, I removed it by mistake. Now should be fine :)